x/cork/keeper/keeper_test.go

```diff
@@ -37,126 +32,10 @@ func TestCellarIDs_SetGetHas(t *testing.T) {
 	})
 }
 
-func TestSetCorkGetCork_Unit(t *testing.T) {
-	testCases := []struct {
-		name string
-		test func()
-	}{
-		{
-			name: "set cork and get cork - happy",
-			test: func() {
-				cellarID := sampleCellarAddr
-				valCork := types.ValidatorCork{
-					Validator: sampleValAddr.String(),
-					Cork: &types.Cork{
-						TargetContractAddress: cellarID.String(),
-						EncodedContractCall:   []byte{33},
-					},
-				}
-
-				k, ctx, _, _ := setupCorkKeeper(t)
-
-				valAddr, err := sdk.ValAddressFromBech32(valCork.Validator)
-				require.NoError(t, err)
-
-				t.Log("Verify no cork exists for the validator")
-				cork, found := k.GetCork(ctx, valAddr, sampleCellarAddr)
-				assert.Equal(t, false, found)
-				assert.Error(t, cork.ValidateBasic())
-				assert.Contains(t, cork.ValidateBasic().Error(),
-					"cork has an empty contract call body")
+func TestSetSecheduledCorkGetSecheduledCCork_Unit(t *testing.T) {
```

Comment:
With the unit tests being empty here now, we should repopulate them appropriately.